domains/chris8889.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "testing",
+    "owner": {
+      "username": "Chris",
```

Comment:
not your username?

Reply:
Oh, I didn't know it had to be my github username. I changed it